src/server/auditLog.ts

```diff
@@ -0,0 +1,61 @@
+"use server";
+
+import { getCurrentUser } from "@/server/getCurrentUser";
+import { InsertSchema } from "@/databaseUtils";
+import { logErrorReturnLogId } from "@/logger/logger";
+import { DatabaseError } from "@/app/errorClasses";
+import { Json } from "@/databaseTypesFile";
+import { getSupabaseServerComponentClient } from "@/supabaseServer";
+
+type AuditLogInsertRecord = InsertSchema["audit_log"];
+export interface AuditLogProps {
+    action?: string;
+    content: Json | null;
+    wasSuccess: boolean;
+    logId?: string;
+    clientId?: string;
+    collectionCentreId?: string;
+    eventId?: string;
+    familyMemberId?: string;
+    listId?: string;
+    listHotelId?: string;
+    packingSlotId?: string;
+    parcelId?: string;
+}
+
+export async function sendAuditLog(auditLogProps: AuditLogProps): Promise<void> {
```

Comment:
nitpick: I don't think the name `prop` is needed here
```suggestion
export async function sendAuditLog(auditLog: AuditLog): Promise<void> {
```